### PR TITLE
Adding LocationComponent customization code snippet

### PR DESCRIPTION
**Pull request by langsmith**
This pr adds some guiding code snippets for customizing the Maps SDK's `LocationComponent`. The other page's other sections have helpful code, but the `Customization` section was missing some code.

screen shot 2018-12-03 at 2 42 47 pm
screen shot 2018-12-03 at 2 42 44 pm

### CODE REVIEW

**Comment by LukasPaczos** [comment]
LGTM :round_pushpin:

File: src/pages/maps/overview/location-component.md [hunk 1]
```diff
@@ -64,9 +66,46 @@ locationComponent.isLocationComponentEnabled = true
...
+The `LocationComponent` allows for several customizations. You can set the drawables, opacities, colors, and more. Create a [LocationComponentOptions](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponentOptions.java) object and then use whichever methods you'd like. Then use the object either while activating the component or by passing it through as a parameter of the `LocationComponent#applyStyle()` method at a later time.
+
+[Here's a list of all of the attributes that can be customized](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/res-public/values/public.xml#L109-L154) via the `LocationComponent`'s methods.
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/location-component.md]: Per the documentation style guide:

>Information about why a link might be of interest should precede the link.

so maybe rephrase: 

>See all the attributes that can be customized via the `LocationComponent`'s methods in the [full list of attributes and styles that are part of the public API](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/res-public/values/public.xml#L109-L154).

Comment (langsmith) [reply src/pages/maps/overview/location-component.md]: Fixed
